views/scss/inc/_action-bars.scss

```
@@ -75,7 +81,26 @@
                 width: 24px;
                 height: 24px;
                 line-height: 24px;
-                text-align: center
+                text-align: center;
+            }
+            .icon-find {
+                padding: 0px;
+                cursor: pointer;
+                width: 25px;
+            }
+            .icon-ul {
+                cursor: pointer;
+                border-left: 1px solid $grey;
+                display: none;
+                margin: 0px;
+                padding: 0px 0px 0px 5px;
```

unit must not be supplied for `0` value
```suggestion
                margin: 0;
                padding: 0 0 0 5px;
```